src/ctia/auth/jwt.clj

```diff
@@ -72,6 +71,11 @@
   (get-in-config [:ctia :auth :casebook :scope]
                               "casebook"))
 
+(defn assets-root-scope [get-in-config]
+  (if (= "private-intel" (entity-root-scope get-in-config))
```

Comment:
I'm not sure I understand why you require this conditional statement, IMO it should be the same code as `casebook-root-scope` but for `asset`

Reply:
Hmm. As you stated yourself in the https://github.com/threatgrid/iroh/issues/4677

> In a second step, exclude all Assets capabilities from the private-intel scope.

So I thought: to have asset capabilities one has to have both "non-private-intel" scope and "asset-intel" scope. No?

Reply:
haha no, for example one could have both the `private-intel` scope and the `asset` scope and be allowed to access everything at once, it's likely what's going to happen anyway.

Reply:
Okay, I'm revisiting the PR and I think I'm still confused. 

My current understanding is:
- if user has `asset-intel` scope  - they should have all asset enabled capabilities
- if user has `private-intel` scope, but doesn't have `asset-intel` scope - they shouldn't have access to asset enable capabilities

Which simply seems to be boiling down to: 
- if user doesn't have `asset-intel` scope - they shouldn't have access to asset capabilities
- `asset-intel` scope enabled by default
- `asset-intel` scope is **not**  enabled by default if `private-intel` scope is present, for a user with `private-intel` scope to have access to asset capabilities, they must include `asset-intel` scope explicitly.

Does that look accurate? Or am I still confused?

@ereteog

Reply:
>     * if user has `asset-intel` scope  - they should have all asset enabled capabilities
> 
>     * if user has `private-intel` scope, but doesn't have `asset-intel` scope - they shouldn't have access to asset enable capabilities

Exactly


> Which simply seems to be boiling down to:
> 
>     * if user doesn't have `asset-intel` scope - they shouldn't have access to asset capabilities
> 
>     * `asset-intel` scope enabled by default
> 
>     * `asset-intel` scope is **not**  enabled by default if `private-intel` scope is present, for a user with `private-intel` scope to have access to asset capabilities, they must include `asset-intel` scope explicitly.
> 

I am not sure that 3 bullet points is a summary of 2 bullet points :-).

Let's keep it simple: asset related routes require `asset-intel` scope.
==>
```clojure
(defn assets-root-scope [get-in-config]
    (get-in-config [:ctia :auth :assets :scope] "asset-intel"))

Reply:
Addressed in a410e79162a6a3cf5d7836261d8d883485269977

Beware though. We're going to have to explicitly disable asset root scope, if we don't want to expose Asset capabilities.